NitroxServer/GameLogic/Bases/BuildingManager.cs

```diff
@@ -102,10 +102,24 @@ public bool ModifyConstructedAmount(ModifyConstructedAmount modifyConstructedAmo
             Log.Error($"Trying to modify the constructed amount of a non-registered object (GhostId: {modifyConstructedAmount.GhostId})");
             return false;
         }
+        // Certain entities are just "regular" WorldEntities and for simplicity we'll just ignore the 
         if (entity is not GhostEntity && entity is not ModuleEntity)
         {
-            Log.Error($"Trying to modify the constructed amount of an entity that is not a ghost (Id: {modifyConstructedAmount.GhostId})");
-            return false;
+            // In case the entity was fully deconstructed
+            if (modifyConstructedAmount.ConstructedAmount == 0f)
```

Comment:
Should this check be `<= 0f` to account for floating point errors?

Reply:
Amount is clamped like so:
```cs
float amount = NitroxModel.Helper.Mathf.Clamp01(constructable.constructedAmount);
```
Thus I believe there's no reason to account for values < 0 and >1 after the network deserialize those